servlet/src/main/java/io/undertow/servlet/spec/ServletContextImpl.java

```diff
@@ -296,7 +296,7 @@ public String getMimeType(final String file) {
 
     @Override
     public URL getResource(final String path) throws MalformedURLException {
-        if (!path.startsWith("/")) {
+        if (path == null || !path.startsWith("/")) {
```

Comment:
According to javadoc if path argument is null, we should return null.

Reply:
This is part of "inconsistency" of this API that I mentioned last week.
JDOC: "The path must begin with a / and is interpreted " - null does not start with "/".

Reply:
Got it.